### ISSUE TITLE
DeDRM fails to decrypt Adobe Digital Editions epub due to `ValueError: string argument should contain only ASCII characters`

### ISSUE THREAD

**Issue by willcallender** [noDRM/DeDRM_tools#402]
I'm attempting to strip the DRM off an EPUB downloaded by Adobe Digital Editions v4.5.11.187303. I've setup DeDRM following multiple online tutorials, and I seem to have successfully installed the plugin and configured with my Adobe keys. I get the following debug log when I try to import my EPUB with Calibre in debug mode.

```
ApplicationPaletteChange event ignored
Using calibre Qt style: True
calibre Debug log
calibre 6.26  embedded-python: True
Windows-10-10.0.22621-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.22621')
Python 3.10.1
Windows: ('10', '10.0.22621', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (10, 0, 3)
calibre 6.26  embedded-python: True
Windows-10-10.0.22621-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.22621')
Python 3.10.1
Windows: ('10', '10.0.22621', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (10, 0, 3)
QPA platform: windows
devicePixelRatio: 1.0
logicalDpi: 96.0 x 96.0
physicalDpi: 95.6235294117647 x 95.58188153310104
[0.00] Starting up...
[0.00] Showing splash screen...
[0.19] splash screen shown
[0.19] Initializing db...
[0.19] db initialized
[0.19] Constructing main UI...
[0.69] main UI initialized...
[0.69] Hiding splash screen
Starting QuickView
DeDRM v10.0.3: Trying to decrypt Evicted.epub
DeDRM v10.0.3: Verifying zip archive integrity
DeDRM v10.0.3: Evicted.epub is a secure Adobe Adept ePub for UUID 73d71589-d114-460a-b7ce-4f03ba40c593
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "encodings\hex_codec.py", line 19, in hex_decode
ValueError: string argument should contain only ASCII characters

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "calibre\customize\ui.py", line 187, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 1067, in run
  File "calibre_plugins.dedrm.__init__", line 533, in ePubDecrypt
ValueError: decoding with 'hex' codec failed (ValueError: string argument should contain only ASCII characters)
[41.97] splash screen hidden
[41.97] Started up in 41.97 seconds with 1 books
Added Evicted: Poverty and Profit in the American City to db in: 0.1
Added 1 books in 0.4 seconds
```

**Comment by willcallender** [comment]
Sorry, this is a duplicate of #388

**Comment by ElleKayEm** [comment]
Fixed in 10.0.9.
